### PR TITLE
Add rubygems user stats

### PR DESCRIPTION
**Pull request by raphink**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1185,6 +1185,44 @@ cache(function(data, match, sendBadge, request) {
...
+      if (count === 0) {
+        badgeData.colorscheme = 'red';
+      } else if (count < 10) {
+        badgeData.colorscheme = 'yellow';
+      } else if (count < 50) {
+        badgeData.colorscheme = 'yellowgreen';
+      } else if (count < 100) {
+        badgeData.colorscheme = 'green';
+      } else {
+        badgeData.colorscheme = 'brightgreen';
```
Comment (espadrine) [inline server.js]: Want to use your fancy new `floorCountColor()` function? ;)

File: server.js [hunk 2]
```diff
@@ -1185,6 +1185,44 @@ cache(function(data, match, sendBadge, request) {
...
+      }
+      badgeData.text[1] = count+' gem';
+      if (count > 1) {
+        badgeData.text[1] += 's';
```
Comment (espadrine) [inline server.js]: I think adding `gem` at the end is redundant, since it's already in the label.

File: server.js (hunk 1 above)
Comment (raphink) [inline server.js]: Yes I was waiting for the other PR to be merged first to avoid depending on it.

File: server.js (hunk 2 above)
Comment (raphink) [inline server.js]: Right, but then how do viewers know is it's downloads or number it's gems?

**Comment by raphink** [comment]
@espadrine fixed both comments.

**Pull request merged**